dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/logger/LogcatLogHandler.kt

```diff
@@ -49,5 +92,12 @@ internal class LogcatLogHandler(
             Log.ERROR to "E",
             Log.ASSERT to "A"
         )
+
+        private val ANONYMOUS_CLASS =
+            Pattern.compile("(\\$\\d+)+$")
+        private const val MAX_TAG_LENGTH = 23
```

Comment:
Is this something copy pasted from Timber ? Why 23 ?

Reply:
It is from Timber....as I understood there's a max length of 23 allowed for the TAG in Logcat for versions lower than N